### DISCUSSION TITLE
Persistent Settings

### DISCUSSION THREAD

**Discussion by davidakerr** [open-webui/open-webui#1537] [Ideas]
Perhaps I am misconfigured, but it seems that user settings default models, API keys for OPEN AI KEYS etc are stored on a per browser instance basis. Ideally these would be stored against the user in the DB.

![image](https://github.com/open-webui/open-webui/assets/31540828/5d344ee8-b1fc-48ea-aa91-8a09bdca1c98)

**Comment by justinh-rahb** [comment]
OpenAI keys are stored/used globally, so there'd be no way for this to be a per-user setting as the current codebase stands.

**Reply by davidakerr** [reply]
As it stands there are additional options such as default model or separate image generation API keys that need to be set per browser. In my cases at least.
